### DISCUSSION TITLE
Ant Media @antmedia/webrtc_adaptor with Angular

### DISCUSSION THREAD

**Discussion by MBFLYT** [ant-media/Ant-Media-Server#5735] [Q&A] [answered]
![image](https://github.com/ant-media/Ant-Media-Server/assets/129502950/959b6a88-aa9a-413b-affd-39dafc1db9f5)
![image](https://github.com/ant-media/Ant-Media-Server/assets/129502950/53f9ae39-035a-48a8-b470-9c41fe247ea4)

Whenever I try to enable the audio level using the webRTCAdapter it tries to load file with this path, is this an known issue.

Please advice.

**Comment by USAMAWIZARD** [comment]
There is an another way to get the audio level , I think this one is deprecated let me confirm this and write you here soon

**Reply by MBFLYT** [reply]
Thank you for the response @USAMAWIZARD,  I am actually following the conference.html and any help from your side is appreciated.

**Reply by MBFLYT** [reply]
Hey @USAMAWIZARD, have you found the way to get the audio level?

**Reply by USAMAWIZARD** [reply, marked as answer]
Please update webrtc_adaptor adapter to latest 

Please make sure you are using latest version of Ant Media Server 

`npm i @antmedia/webrtc_adaptor@2.7.0-SNAPSHOT-2023-Oct-21-01-09`

[Here](https://github.com/ant-media/conference-call-application/blob/97c496ea053afd171363f051de8a573ec78ee37e/react/src/pages/AntMedia.js#L1316) is the new way to do that but it is released in latest version of Ant Media Server.

**Comment by lastpeony** [comment]
This method for retrieving sound levels is no longer supported.

We recommend upgrading your Ant Media Server to the latest version, utilizing the latest SDK, and adopting the use of 'multitrack-conference.html.'

In the new version, the sound levels of conference participants are transmitted from the server side to each client through the data channel.

**Reply by MBFLYT** [reply]
Sorry for the delay, @lastpeony and @USAMAWIZARD  the solution you had provided had worked very well thanks for the guidance.

**Reply by USAMAWIZARD** [reply]
@MBFLYT  I would be very happy if you can post a review for us on AWS. 

Please note the following points while adding a review otherwise it gets rejected By AWS.

1. AI-generated content.
2. Users not providing their real names.
3. Duplicate reviews.

[1] https://aws.amazon.com/marketplace/pp/prodview-464ritgzkzod6?sr=0-1&ref_=beagle&applicationId=AWSMPContessa#pdp-reviews

Thanks in advance.
